src/WebJobs.Script.WebHost/App_Start/WebApiConfig.cs

```diff
@@ -109,28 +94,28 @@ private static void PrependFoldersToEnvironmentPath()
                 var folders = new List<string>();
                 folders.Add(Path.Combine(home, @"site\tools"));
 
-                string path = _settingsManager.GetSetting("PATH");
+                string path = Environment.GetEnvironmentVariable("PATH");
```

Comment:
Decided to switch this back as the intent here is really to update the environment path and wouldn't make sense to push this through another implementation of the settings manager as the functionality would be broken.